capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/main.js

```diff
@@ -321,10 +321,6 @@ angular.module('c4mApp')
       // Check for required fields.
       var errors = Request.checkRequired(submitData, resource, resourceFields);
 
-      // Check the type of the submit.
-      // Make node unpublished if requested to create in full form.
-      submitData.status = type == 'full_form' ? 0 : 1;
```

Comment:
Since we removed tab Published/unpublished - we can't make the node published - so it should be or published from the very beginning - or we should give to user an option to make it published after he finishes creation.